projects/core/components/primitive-checkbox/primitive-checkbox.style.less

```diff
@@ -1,22 +1,25 @@
 @import '../../styles/taiga-ui-local.less';
 
-@sizeM: 16px;
-@sizeL: 24px;
-
 :host {
     display: block;
     border-radius: 8px;
     font-size: 0;
 
+    &[data-tui-host-size='s'] {
+        width: calc(var(--tui-height-s) / 2);
+        height: calc(var(--tui-height-s) / 2);
+        border-radius: 4px;
+    }
+
     &[data-tui-host-size='m'] {
-        width: @sizeM;
-        height: @sizeM;
+        width: calc(var(--tui-height-m) / 2);
+        height: calc(var(--tui-height-m) / 2);
         border-radius: 4px;
     }
 
     &[data-tui-host-size='l'] {
-        width: @sizeL;
-        height: @sizeL;
+        width: calc(var(--tui-height-l) / 2);
```

Comment:
I noticed inconsistency in size variables and removed 40px in last commit, moving 44 to m and 56 to l, dropping 40px and xl naming. But even before that these sizes were wrong. Checkboxes have to be 16 and 24px in size. Do you feel like we could use a third size for checkboxes?

Reply:
This is of cause a very personal decision. We are looking for a size of 20px in our design system, but I guess we could also try and adjust. I found it a nice idea to have checkboxes react to the new height variables as well and just set them to half of these values. This obviously clashes with the recent changes of m=44 and l=56.